packages/eslint-plugin/lib/utils/patterns.ts

```diff
@@ -1,5 +1,6 @@
 export const sourceFiles = [
-    "*.[jt]s"
+    "*.[jt]s?(x)",
+    "*.[cm]js"
```

Comment:
added this just in case, as they are not included in the previous regex, but should still be linted